dev/com.ibm.ws.jaxws.2.3.clientcontainer/src/com/ibm/ws/jaxws/client/LibertyServiceImpl.java

```diff
@@ -144,13 +147,12 @@ protected void configureCustomizeBinding(Client client, QName portName) {
                 requestContext.putAll(portProps);
             }
 
-//            Commented out to be solved later by replacing LoggingFeature (cxf/rt/features/logging)
-//            if (null != wsrProps && Boolean.valueOf(wsrProps.get(JaxWsConstants.ENABLE_lOGGINGINOUTINTERCEPTOR))) {
-//                List<Interceptor<? extends Message>> inInterceptors = client.getInInterceptors();
-//                inInterceptors.add(new LoggingInInterceptor());
-//                List<Interceptor<? extends Message>> outInterceptors = client.getOutInterceptors();
-//                outInterceptors.add(new LoggingOutInterceptor());
-//            }
+            if (null != wsrProps && Boolean.valueOf(wsrProps.get(JaxWsConstants.ENABLE_lOGGINGINOUTINTERCEPTOR))) {
```

Comment:
We need to comment this back out again because we shouldn't be using these interceptors with CXF 3.3